### PR TITLE
Change img url path

### LINKED ISSUE #85: Change img url path, filter logo, interfaces
_change img url path (care-body header)
_change style
_add api constant (share/constatnts)
_fix type interface ( remove | undefined)
_add new logo (filter page)

### PR DESCRIPTION
**Pull request by Pashkaua**
_change img url path (care-body header)
_change style
_add api constant (share/constatnts)
_fix type interface ( remove  | undefined)
_add new logo (filter page)

### CODE REVIEW

**Review by ivan-bonk — APPROVED** [review]
looks good

File: src/pages/filter/components/filter-header/filter-header.component.tsx [hunk 1]
```diff
@@ -1,12 +1,13 @@
 import React from 'react';
-import Logo from 'images/paw-solid.svg';
 import styles from './filter-header.module.scss';
+import { Logo } from '../../../../shared/logo/logo.component';
```
Comment (enheit) [inline src/pages/filter/components/filter-header/filter-header.component.tsx:3]: Use alias import, if it is available

**Review by enheit — APPROVED** [review]

**Review by Danik1702 — APPROVED** [review]
Nice!

File: src/pages/filter/components/filter-header/filter-header.component.tsx (hunk 1 above)
Comment (Pashkaua) [reply src/pages/filter/components/filter-header/filter-header.component.tsx:3]: Оk, i change this in next pull requests.

**Pull request merged**